.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '3.6'
+
+services:
+  - docker
+
+before_install:
+  - docker build -t datawig-sagemaker .
+  - docker images
+#  - docker run
```

Comment:
Can we remove the commented lines or are they intended as some kind of documentation?